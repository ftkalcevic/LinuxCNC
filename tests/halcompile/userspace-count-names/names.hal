loadusr -W userspace_count_names names=bibble,bobble
list pin
