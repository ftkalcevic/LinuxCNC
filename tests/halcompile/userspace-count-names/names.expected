count=0
names[0]=bibble
names[1]=bobble
argc=1
argv[0]=userspace_count_names
argv[1]=(null)
bibble.out
bobble.out

